pyro/mesh/array_indexer.py

```diff
@@ -281,7 +281,7 @@ def pretty_print(self, n=0, fmt=None, show_ghost=True):
         """
 
         if fmt is None:
-            if self.dtype == np.int:
```

Comment:
These should check against `np.int64`, or maybe `isinstance(self.dtype.type, numbers.Integral)` for better compatibility (from https://stackoverflow.com/a/62055141)